birdhouse/default.env

```diff
@@ -80,6 +80,24 @@ export SERVER_RELEASE_NOTES_URL='${__DEFAULT__SERVER_RELEASE_NOTES_URL}'
 export SERVER_SUPPORT_URL='${__DEFAULT__SERVER_SUPPORT_URL}'
 export SERVER_LICENSE_URL='${__DEFAULT__SERVER_LICENSE_URL}'
 
+# Defaults for required variables recommended for override for security reasons.
+# Those will not be set explicitly as defaults to ensure they are overridden explicitly by the instance.
+# These values would be detected only if the instance was configured using a copy of 'env.local.example'.
+export __DEFAULT__MAGPIE_SECRET=itzaseekrit
+#export __DEFAULT__MAGPIE_ADMIN_USERNAME=admin
+export __DEFAULT__MAGPIE_ADMIN_PASSWORD=qwertyqwerty!
+#export __DEFAULT__POSTGRES_PAVICS_USERNAME=postgres-pavics
+export __DEFAULT__POSTGRES_PAVICS_PASSWORD=postgres-qwerty
+#export __DEFAULT__POSTGRES_MAGPIE_USERNAME=postgres-magpie
+export __DEFAULT__POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
+#export __DEFAULT__GEOSERVER_ADMIN_USER=admingeo
+export __DEFAULT__GEOSERVER_ADMIN_PASSWORD=geoserverpass
+export __DEFAULT__TOMCAT_NCWMS_PASSWORD=ncwmspass
```

Comment:
I think all the variables at this line and below are for deprecated components. Can you add in a comment saying that so that in the future they're easy to identify as deprecated?

Reply:
yes